src/components/tripDetails/_Tooltip.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+	/*
+    IMPORTANT NOTE:
+
+    ----Work in progress----
+
+    Müssen wir nochmal überarbeiten
```

Comment:
Tatsächlich etwas buggy, kann ich später gerne in Angriff nehmen, ich wollts ja immerhin auch. :D Aber soll den PR erstmal nicht mehr blocken